spec/jsSpecs/ares-shell.spec.js

```diff
@@ -4,16 +4,20 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+/* eslint-disable no-useless-escape */
 const exec = require('child_process').exec,
+    os = require('os'),
     common = require('./common-spec');
 
 const aresCmd = 'ares-shell';
 
 let cmd,
     options,
-    hasSession = false;
+    hasSession = false,
+    osType;
 
 beforeAll(function (done) {
+    osType = os.type();
```

Comment:
Why don't you use os.platform or process.platform(), which is already used in CLI ?

Reply:
I've updated this point